packages/wdio-cli/src/utils.js

```diff
@@ -196,3 +197,38 @@ export function addServiceDeps(names, packages, update) {
         }
     }
 }
+
+export function getCapabilities(arg) {
+    const capabilities = getPlatformCaps(arg.option)
+    let finalCapabilities = capabilities
+    if (capabilities.platformName === IOS_CONFIG.platformName) {
+        finalCapabilities = Object.assign( {},
+            capabilities,
+            {
+                deviceName: arg.device || IOS_CONFIG.deviceName,
+                platformVersion: arg.ver || null,
+                udid: arg.udid || null
+            }
+        )
+    }
+    return finalCapabilities
+}
+
+function getPlatformCaps(option) {
+    let platformConfig = {}
+    switch (option) {
+    case /.*\.(apk|app|ipa)$/.test(option) && option:
+        platformConfig = Object.assign({ app: option }, option.endsWith('apk') ? ANDROID_CONFIG : IOS_CONFIG)
+        break
+    case 'android':
+        platformConfig = Object.assign({ browserName: 'Chrome' }, ANDROID_CONFIG)
+        break
+    case 'ios':
+        platformConfig = Object.assign({ browserName: 'Safari' }, IOS_CONFIG)
+        break
+    default:
+        platformConfig = { browserName: option }
```

Comment:
Instead of reassigning `platformConfig` and then returning it, you could just return the assignment from the switch / case statement. Also instead of using `Object.assign` you could just destructure the object, but this is more a matter of preferability.

Example:

```js
case /.*\.(apk|app|ipa)$/.test(option) && option:
    return {
        app: option,
        ...(option.endsWith('apk') ? ANDROID_CONFIG : IOS_CONFIG)
        // or ...ANDROID_CONFIG / ...IOS_CONFIG for the other 2 cases
    }
```